x-pack/plugins/case/server/services/connector_mappings/index.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  Logger,
+  SavedObject,
+  SavedObjectReference,
+  SavedObjectsClientContract,
+  SavedObjectsFindResponse,
+  SavedObjectsUpdateResponse,
+} from 'kibana/server';
+
+import { ConnectorMappings, SavedObjectFindOptions } from '../../../common/api';
+import { CASE_CONNECTOR_MAPPINGS_SAVED_OBJECT } from '../../saved_object_types';
+
+interface ClientArgs {
+  client: SavedObjectsClientContract;
+}
+
+interface GetConnectorMappingsArgs extends ClientArgs {
+  caseConfigureId: string;
+}
+interface FindConnectorMappingsArgs extends ClientArgs {
+  options?: SavedObjectFindOptions;
+}
+
+interface PostConnectorMappingsArgs extends ClientArgs {
+  attributes: ConnectorMappings;
+  references: SavedObjectReference[];
+}
+
+interface PatchConnectorMappingsArgs extends ClientArgs {
+  caseConfigureId: string;
+  updatedAttributes: Partial<ConnectorMappings>;
+}
+
+export interface ConnectorMappingsServiceSetup {
+  delete(args: GetConnectorMappingsArgs): Promise<{}>;
+  get(args: GetConnectorMappingsArgs): Promise<SavedObject<ConnectorMappings>>;
+  find(args: FindConnectorMappingsArgs): Promise<SavedObjectsFindResponse<ConnectorMappings>>;
+  patch(args: PatchConnectorMappingsArgs): Promise<SavedObjectsUpdateResponse<ConnectorMappings>>;
+  post(args: PostConnectorMappingsArgs): Promise<SavedObject<ConnectorMappings>>;
+}
```

Comment:
@XavierM @cnasikas I made this based on other SO services, but right now we're actually only using `find` and `post`. Do you agree that I should just make all of the methods, or should I only make the 2 I'm using?

Reply:
I think is best to expose only the one we are using at the moment.